app/perfunwind.h

```diff
@@ -279,6 +288,21 @@ class PerfUnwind : public QObject
 
     QList<PerfRecordSample> m_sampleBuffer;
     QList<PerfRecordMmap> m_mmapBuffer;
+
+    // Cache Entry containing follow up addresses, used for stack stitching
+    struct OffsetFrames {
+        // QVector is COW so we only have one instance
+        QVector<int> frames;
+        int framesOffset = 0;
+
+        void appendFrames(QVector<int>& target) const {
+            target.reserve(target.size() + frames.size() - framesOffset);
+            std::copy(frames.begin() + framesOffset, frames.end(), std::back_inserter(target));
+        }
+    };
+
+    QHash<int, QHash<quint64, OffsetFrames>> m_ipcache;
```

Comment:
capitalize: `m_ipCache`, and add a comment that indicates what this maps:

pid to address to frames